pages/room/[id].js

```diff
@@ -48,6 +48,26 @@ export default function Room(props) {
           )
         })}
       </div>
+      <div className="flex justify-center">
+        <button
+          type="submit"
+          className="w-1/5 m-5 font-display text-white bg-[#26374A] hover:bg-[#1C578A] active:bg-[#16446C] focus:bg-[#1C578A] py-2 px-2 rounded border border-[#091C2D] text-[16px] leading-8"
+        >
+          {t.showButton}
+        </button>
+        <button
+          type="submit"
+          className="w-1/5 m-5 font-display text-white bg-[#26374A] hover:bg-[#1C578A] active:bg-[#16446C] focus:bg-[#1C578A] py-2 px-2 rounded border border-[#091C2D] text-[16px] leading-8"
+        >
+          {t.hideButton}
+        </button>
+        <button
+          type="submit"
```

Comment:
maybe it's a react thing I'm not sure about, but typically you only want to use `submit` type when form data is being passed back, alternatively you can use `button` or `reset` types.